### PR TITLE
Document ES5 shims requirement

### PR DESCRIPTION
**Pull request by mightyiam**

### CODE REVIEW

**Comment by mightyiam** [comment]
Please only review and do not merge due to #679.

File: docs/getting_started/setup.rst [hunk 1]
```diff
@@ -59,9 +59,9 @@ See :ref:`Include jQuery` for WYMeditor's range of supported jQuery versions.
 Source the WYMeditor Javascript
 ===============================
 
-Include the ``wymeditor/jquery.wymeditor.min.js`` file
+Include the dependencies
```
Comment (winhamwr) [inline docs/getting_started/setup.rst]: dependencies (e.g. ES5 shim and ES5 Sham)

File: docs/getting_started/setup.rst [hunk 2]
```diff
@@ -70,6 +70,9 @@ This file will pull in anything else that's required.
             <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
             <title>WYMeditor Quickstart</title>
             <script type="text/javascript" src="jquery/jquery.js"></script>
+            <!--for IE7 & IE8, ES5 shims are required-->
```
Comment (winhamwr) [inline docs/getting_started/setup.rst]: We should use [conditional comments](http://www.quirksmode.org/css/condcom.html) to only include them for Less than IE10.

**Comment by winhamwr** [comment]
If these are a requirement, then we would need them in all of the examples and tests, right?

**Comment by winhamwr** [comment]
Needs:

* [ ] Conditional comment example
* [ ] Conditional shim in unit tests
* [ ] Conditional shim in examples
* [ ] Conditional shim in other tests
* [ ] Backwards incompatible changes section with link to explanation for fixing

Once you have the change notification, I can test how good of a job it does by handing it to one of the PolicyStat engineers. If it gives them the information they need to do the upgrade, it's good enough. If they're confused, we need to help.

File: docs/getting_started/setup.rst (hunk 2 above)
Comment (mightyiam) [inline docs/getting_started/setup.rst]: Thank you.

**Comment by mightyiam** [comment]
I see no compelling reason to make es5-shim conditional in the development environment.

This means unit tests.

**Comment by winhamwr** [comment]
> make es5-shim conditional in the development environment.

By development environment, are you referring to `load-src-dev.js`?

**Comment by mightyiam** [comment]
> > make es5-shim conditional in the development environment.

> By development environment, are you referring to `load-src-dev.js`?

Loading the shims via `load-src.dev.js` saves us adding their script tags to all of the test html files (the primary is the unit test index).

Where we must add the shims' script tags is in the examples, because they are run from the website, where they are built to not use `load-src.dev.js`.

So that's what I did.

File: CHANGELOG.md [hunk 3]
```diff
@@ -35,6 +35,11 @@ WYMeditor.
     tested.
 * [#679](https://github.com/wymeditor/wymeditor/pull/679)
   Support for IE7 dropped.
+* [#680](https://github.com/wymeditor/wymeditor/pull/680)
+  For IE8 support, some shimmable ES5 features are now required.
+  We test using [es5-shim](https://github.com/es-shims/es5-shim)
+  (including the shams).
+  Here are [examples](https://github.com/wymeditor/wymeditor/pull/680/files).
```
Comment (mightyiam) [inline CHANGELOG.md]: Is this enough?

File: docs/getting_started/setup.rst [hunk 4]
```diff
@@ -70,6 +70,11 @@ This file will pull in anything else that's required.
             <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
             <title>WYMeditor Quickstart</title>
             <script type="text/javascript" src="jquery/jquery.js"></script>
+            <!-- for IE8, ES5 shims are required-->
```
Comment (winhamwr) [inline docs/getting_started/setup.rst]: Aren't they also required for IE9?

**Comment by winhamwr** [comment]
Looks good to me, assuming those examples work in IE9. I remember reading that IE10 was the first version with good ES5 support, though.

**Comment by winhamwr** [comment]
> Loading the shims via load-src.dev.js saves us adding their script tags to all of the test html files (the primary is the unit test index).

:+1:

File: docs/getting_started/setup.rst (hunk 4 above)
Comment (mightyiam) [inline docs/getting_started/setup.rst]: Nope. IE9 runs pretty good ES5.

**Comment by mightyiam** [comment]
@winhamwr Just FYI:
## IE9
ie9
## IE10
ie10

**Pull request merged**